src/Components/CreateImageWizard/utilities/useValidation.tsx

```diff
@@ -341,17 +344,26 @@ export function useUsersValidation(): StepValidation {
     users.length === 0 ||
     // Case 2: All fields are empty
     (userName === '' && userPassword === '' && userSshKey === '') ||
-    // Case 3: userName is valid and SshKey is valid
+    // Case 3: userName is valid and SshKey or Password is valid
     (userName &&
       isUserNameValid(userName) &&
-      userSshKey &&
-      isSshKeyValid(userSshKey));
+      ((userSshKey && isSshKeyValid(userSshKey)) ||
+        (userPassword && isPasswordValid(userPassword))));
 
   return {
     errors: {
-      userName: !isUserNameValid(userName) ? 'Invalid user name' : '',
+      userName: !userName
+        ? 'default'
+        : !isUserNameValid(userName)
+        ? 'Invalid user name'
+        : '',
+      userPassword: !userPassword
+        ? 'default'
+        : !isPasswordValid(userPassword)
+        ? 'Password must be between 8 and 128 characters, contain at least one uppercase letter, one lowercase letter, and one special character.'
+        : '',
```

Comment:
The default values shouldn't be added here, but to the `ValidationInput` component. Will add a comment there